src/users/controllers/CreateUserController.js

```diff
@@ -0,0 +1,32 @@
+import { Prisma } from '@prisma/client';
+import { prismaClient } from '../../../database/prismaClient.js';
+import { encryptPassword } from  '../utils/encrypt.js';
+
+export class CreateUserController {
+    async handle(request, response) {
+        const {name, email, password} = request.body;
```

Comment:
Try posting from postman but with some missing field and see what happens if you use destructing for the body’s content. I recommend some sort of validation library instead, perhaps https://www.npmjs.com/package/joi

Reply:
Eu vi algumas bibliotecas de validação. 
Vi a express-validator, JOI e a ajv!
Vou adicionar no meu fluxo. 
Eu vi que o Joi ele parte de um schema, estou usando prisma, tem como eu sincronizar o Joi com Prisma?
Acredito que o `express-validator` seria o mais simples de implementar

Reply:
Não manjo alguma que consiga usar o schema do prisma, talvez até tenha.

Qual lib usar meio que tanto faz, usa a que vc achar que te atende melhor. O ponto é que se vc usar destructing em uma propriedade que não existe vai dar ruim. Então só use destructing em propriedade que vc sabe com certeza que existe. Daí a necessidade de validar a existência dos campos.

De quebra, vc já usa o validador pra validar o valor dos campos e até fazer coerção de tipos quando necessário.

Reply:
Maneiro demais! Valeu @rwillians